### DISCUSSION TITLE
Paid for Github Copilot on april 8th but I get "Your GitHub Copilot subscription has ended."

### DISCUSSION THREAD

**Discussion by Jejugo** [community/community#52928] [Copilot Conversations]
### Select Topic Area

Bug

### Body

As written in the title post, I paid for the github copilot membership and the payment was processed on April 8th. I even have the receipt of the transaction. However, on the github copilot section I get "Your GitHub Copilot subscription has ended.". And even when I start using it on visual code I also get the same message. Can someone assist me in how to resolve this?

**Comment by Akash1134** [comment]
Hi @Jejugo The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by wscnd** [comment]
Hello @Akash1134, I have the same issue as the one described by @Jejugo , I have received "Your GitHub Copilot subscription has ended" messages.
My subscription plan was supposed to be running throughout Feb 14, 2023 - Feb 13, 2024.
I have opened a ticket on your Support page as you instructed.
Thank you!
